app/views/report_templates/jobs_invocations.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+name: Jobs - Invocations
+snippet: false
+template_inputs:
+- name: job_id
+  required: true
+  input_type: user
+  description: ID of job to report
+  advanced: false
+  value_type: plain
+  resource_type: JobInvocation
+  hidden_value: false
+- name: hosts
+  required: false
+  input_type: user
+  description: Field for filter hosts of job invocation. Leave blank for all hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+  hidden_value: false
+model: ReportTemplate
+-%>
+<%- invocation = load_job_invocation_by_id(input('job_id')) -%>
+<%- load_hosts(search: input('hosts'), where: {id: template_invocations_hosts.ids}).each do |host| -%>
+<%-   task = invocation.sub_task_for_host(host) -%>
+<%-   sub_task_output = task.output_by_type -%>
```

Comment:
If I'm reading this right then `task` here is an instance of `ForemanTasks::Task`, but `output_by_type` is defined on `RunHostJob` so `task.output_by_type` should raise `NoMethodError` and we should use something along the lines of `task.main_action.output_by_type`

Reply:
I have two options to solve this problem: 

1) Add macro for `task.main_action.output_by_type`. I know, more macros is not a good idea but I would like to shield user from the internals (sounds funny for OSS :D )

2) Allow `main_action` in safe mode. Is there any drawback for that? My opinion is not to do that. 
 
What is more suitable for you? @adamruzicka @ezr-ondrej @ares

Reply:
Well, given the @adamruzicka's comment above (format of the outputs) we really should rename the method to `output`. And I guess it's ok to add `action_output` to `Jail` enabled methods and it being safeguard as well if the task doesn't have output.

```ruby
# ForemanTasks::Task
def action_output
  main_action.output if main_action.respond_to?(:output) #or maybe if main_action.is_a?(Actions::Helpers::WithContinuousOutput)
end
```

BUT `main_action` is already in `Jail` enabled methods. If we Jail guard action as well, there shouldn't be problem to do that.